week2/project/ecommerce/src/CardSetMain.jsx

```diff
@@ -0,0 +1,18 @@
+import ProductCard from './ProductCard.jsx';
+
+const CardSetMain = ({ filterProducts }) => {
+    return (
+        <div className="card-set">
```

Comment:
```suggestion
        <div className="product-list">
```
Suggestion: 
Also ClassName can be more business meaning instead of UI meanings, could you try to rename (ex: card, button)